src/components/select/select-list/select-list.component.js

```diff
@@ -59,6 +59,8 @@ const SelectList = React.forwardRef(
       multiColumn,
       tableHeader,
       loaderDataRole,
+      listPlacement = "bottom-start",
```

Comment:
The defaults might be better being in the `Select` components so the values show up in the prop tables